docs/source/advanced/bgfx.rst

```diff
@@ -75,34 +75,34 @@ Configuration Settings
 |
 |	On a single window, multiple screen game, such as Darius on one physical PC monitor, specify multiple entries (one per window) like:
 |
-|		**bgfx_screen_chains hlsl,hlsl,hlsl**
+|		``bgfx_screen_chains hlsl,hlsl,hlsl``
 |
 |	This also works with single screen games where you are mirroring the output to more than one physical display. For instance, you could set up Pac-Man to have one unfiltered output for use with video broadcasting while a second display is set up HLSL for playing on.
 |
 |	On a mulitple window, multiple screen game, such as Darius on three physical PC monitors, specify multiple entries (one per window) like:
 |
-|		**bgfx_screen_chains hlsl:hlsl:hlsl**
+|		``bgfx_screen_chains hlsl:hlsl:hlsl``
 |
 |	Another example game would be Taisen Hot Gimmick, which used two CRTs to show individual player hands to just that player. If using two windows (two physical displays):
 |
-|		**bgfx_screen_chains hlsl:hlsl**
+|		``bgfx_screen_chains hlsl:hlsl``
 |
 |	One more special case is that Nichibutsu had a special cocktail mahjongg cabinet that used a CRT in the middle along with two LCD displays to show each player their hand. We would want the LCDs to be unfiltered and untouched as they were, while the CRT would be improved through HLSL. Since we want to give each player their own full screen display (two physical monitors) along with the LCD, we'll go with:
 |
-|		**-numscreens 2 -view0 "Player 1" -view1 "Player 2" -video bgfx -bgfx_screen_chains hlsl,unfiltered,unfiltered:hlsl,unfiltered,unfiltered**
+|		``-numscreens 2 -view0 "Player 1" -view1 "Player 2" -video bgfx -bgfx_screen_chains hlsl,unfiltered,unfiltered:hlsl,unfiltered,unfiltered``
```

Comment:
This one is a command line fragment rather than an INI file fragment.  You could reformat it onto multiple lines without the leading hyphens INI-style or make it bold like other command fragments.

Reply:
Making a note that I should probably actually expand this to show both ways.